docs/developer/Supplemental_developer_guide.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Supplementary Developer Contribution Guidelines
+sidebar_position: 8
+---
+
+# Supplementary Developer Contribution Guidelines:
+
+
+## DCO
+
+  
+DCO (Developer Certificate of Origin) needed for each commit, contributors must sign off on their commits to certify that they have the right
+to submit the code they are contributing. This is done using `--signoff` option in Git.  
+
+The sign-off is a simple line at the end of the commit message, which looks like this:  
+  
+    ```
```

Comment:
![image](https://github.com/user-attachments/assets/3c7e996d-6ff2-4449-8845-803b2c541b05)

Seems some wrong with this format.

Reply:
Updated the format to fix the issue.